### DISCUSSION TITLE
问在启用Bilibili-Evolved时，如何将B站原来的搜索栏恢复成原来的样子

### DISCUSSION THREAD

**Discussion by BitsAstro** [the1812/Bilibili-Evolved#4983] [提问 / Q&A] [answered]
Bilibili-Evolved内置插件里有搜索栏，但因为是内置的所以并没有卸载的按键
所以提问在启用Bilibili-Evolved时，如何将B站原来的搜索栏恢复成原来的样子
求解决办法！

**Comment by Geeyun-JY3** [comment, marked as answer]
Bilibili Evolved 2.x 默认不预装任何组件：https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85
> 注意事项
> ……
> - 默认不预装任何组件

你是装了`自定义顶栏`吧，这是可以卸载的。

**Reply by BitsAstro** [reply]
感谢，确实是，不过“自定义顶栏”这个组件内的插件怎么不支持自定义开关，好奇怪，除了那个搜索栏其他部分我都挺喜欢的

**Reply by Geeyun-JY3** [reply]
怎么不支持，布局设置里就可以关啊：
![image](https://github.com/user-attachments/assets/2991bd06-9413-47ae-9ad1-2b4742811cbc)

**Reply by BitsAstro** [reply]
我去，我以为眼睛那个图标是将整个搜索栏给不显示（隐身掉），好吧误解了，感谢老哥
